source/docs/docker-storage-recommendation.md

```diff
@@ -2,79 +2,130 @@
- 
-    # mkfs.xfs /dev/sdb1
- 
-## Setting Up /var/lib/docker
-
-Before setting up the external volume consider if there are already important images or containers already in use on this host.  It is unlikely due to the size of the image and it is preferred to do these recommended steps before building or pulling down images. i.e. it is not recommended to use Atomic Host before attaching an external volume. But if there is important information in `/var/lib/docker` then it is best to move this to a temporary location until it can be placed on the external volume.
-
-    # mv /var/lib/docker  /my-backupd-dir
-
-Stop the docker service.
-
-    # systemctl stop docker
-
-Now, getting back to the new device, get the UUID from the device.
- 
-    # blkid /dev/sdb1
- 
-Then add that to the `/etc/fstab` in the format:
-
-    UUID=<UUID-from-blkid> /var/lib/docker <file system type> defaults 1 1
- 
-For our xfs example it looks something like like:
- 
-    UUID=be840c0d-91f8-41fa-bb40-82e1c4e2e985 /var/lib/docker xfs defaults 1 1
- 
-
-## Start Using The New Volume With Docker
-
-The volume is ready to be mounted and used by Docker. Mount the device:
-
-    # mount -a
- 
-If you did move any data from `/var/lib/docker` then now is a good time to move it to then new volume. Then restart the docker service:
- 
-    # systemctl start docker
-
-If the provided `/var/lib/docker` was not moved as descrbed above then it is easy to check if everything worked.  After the docker service has started you should see the following files in the `/var/lib/docker` with ls :
-
-    containers  devicemapper  execdriver  graph  init  linkgraph.db  
-    lxc-start-unconfined  repositories-devicemapper  vfs  volumes
-
-If you moved the original contents of `/var/lib/docker` ensure this directory contains the data you expect.
-Another test is to build an image. Here is a simple example `Dockerfile`:
-
-    FROM fedora
-    RUN  yum -y update
-
- 
 ## Looking Forward
 
-In the future it is likely that the recommended mounting for /var/lib/docker will be more granular depending on the performance needs of various sub-directories of `/var/lib/docker`.  Watch this space.
+In order to add a new device using the `docker-storage-setup` script, you will need to make sure that the configuration file only contains references to devices and sizes for this particular run of the tool.  
+
+For example, if you add a new device to host and to the `DEVS` line, the `docker-storage-setup` script will exit as the exisiting device has a partition and physical volume already created.
 
-## Filesystem Considerations
+    [fedora@atomic-host-002 ~]$ sudo vi /etc/sysconfig/docker-storage-setup
+    DEVS="/dev/vdb /dev/vdc"
 
-Docker supports several different filesystem formats. How these work and which one you choose for all or part of your deployment will greatly effect your performance and efficiency.
+     [fedora@atomic-host-002 ~]$ sudo docker-storage-setup
```

Comment:
I am a bit too much detail oriented, but this is not aligned with the other prompt